server/fishtest/views.py

```diff
@@ -530,25 +530,25 @@ def users_monthly(request):
 
 
 def get_master_info():
-    message_search = re.compile(r"^.*$", re.MULTILINE)
-    bench_search = re.compile(r"(^|\s)[Bb]ench[ :]+([0-9]+)", re.MULTILINE)
+    bench_search = re.compile(r"(^|\s)[Bb]ench[ :]+([1-9]\d{5,9})(?!\d)")
     for idx, commit in enumerate(
         requests.get(
```

Comment:
We are doing over and over the same request to GitHub, please move it outside the enumerate loop.

```python
commits = requests.get(........).json()

for idx, commit in enumerate(commits):
```